vulkan_generator/tests/test_types.py

```diff
@@ -23,7 +23,11 @@
 import xml.etree.ElementTree as ET
 
 from vulkan_parser import handle_parser
-from vulkan_parser.types import VulkanHandle, VulkanHandleAlias
+from vulkan_parser import struct_parser
+from vulkan_parser.types import VulkanHandle
+from vulkan_parser.types import VulkanHandleAlias
+from vulkan_parser.types import VulkanStruct
+from vulkan_parser.types import VulkanStructAlias
```

Comment:
https://github.com/google/styleguide/blob/gh-pages/pyguide.md#22-imports

The style guide says not to import individual classes and functions. If we are importing multiple things from the same package though, we should use one line per thing I believe. so instead of:

from package import Thing, OtherThing, ThirdThing

we should use:

from package import Thing
from package import OtherThing
from package import ThirdThing

But consider just 

import package

Unless there's a good reason not to

Reply:
👍

Reply:
```
https://github.com/google/styleguide/blob/gh-pages/pyguide.md#22-imports

The style guide says not to import individual classes and functions. If we are importing multiple things from the same package though, we should use one line per thing I believe. so instead of:

from package import Thing, OtherThing, ThirdThing

we should use:

from package import Thing
from package import OtherThing
from package import ThirdThing

But consider just

import package

Unless there's a good reason not to
```

I think this is not wanted.